.github/workflows/go.yaml

```diff
@@ -0,0 +1,58 @@
+name: go
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Given the latest movements about the naming I wonder if not to start with `main` right away to save some potential work later on :) Just an idea